app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class='recipe-details'>
+  <div class='rd-done'>
+
+    <%= link_to @recipe.name, recipe_path(@recipe), class: 'rd-title' %><br>
+    <span class='rd-username'> by <%= @recipe.user.name %></span>
+  
+    <div class='rd-firstline'>
+      Preparation time: <%= @recipe.preparation_time %> hour
+      <div class='is-public'>
+        <label>Public:<%= check_box_tag 'public', 'no', @recipe.public %><br>
+      </div>
+    </div>
+    
+    <div class='rd-secondline'>
+      Cooking time: <%= @recipe.cooking_time %> hour
+    </div>
+
+    <div class='rd-description'>
+      <p><%= @recipe.description %></p>
+    </div>
+    
+    <div class='rd-two-buttons'>
+      <div class = 'rd-get'>
+        <%= link_to 'Generate shopping list', general_shopping_list_path %>        
+      </div>
```

Comment:
- [ ] Due to the fact that a user is currently unable to add Food items to a given recipe (a bug 🐛 I have highlighted earlier), It is impossible to Generate a general shopping list. This is because for it to calculate and show a list of food that is missing for all recipes, the recipe should have a list of foods. Please fix this 👍 
![image](https://user-images.githubusercontent.com/46839548/208217146-5268208f-dd9e-40f6-801a-b319bed19d9c.png)

Reply:
@codepantha  Now it works. Done !!